cmd/add.go

```diff
@@ -1,129 +1,155 @@
 // Copyright (c) 2023 Cisco Systems, Inc. and its affiliates
 // All rights reserved.
-
 package cmd
 
 import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
-	"net/http" // Import for HTTP requests
-	"os"       // Import for environment variables
+	"net/http"
+	"os"
 	"strings"
 
 	"github.com/cisco-open/grabit/internal"
-	"github.com/rs/zerolog/log" // Import for logging (debug statements)
+	"github.com/rs/zerolog/log"
 	"github.com/spf13/cobra"
 )
 
 func addAdd(cmd *cobra.Command) {
 	addCmd := &cobra.Command{
-		Use:   "add",
+		Use:   "add [url]",
 		Short: "Add new resource",
-		Args:  cobra.MinimumNArgs(1),
+		Args:  cobra.ExactArgs(1),
```

Comment:
This doesn't seem right -- it's permissable to add multiple URL's for a resource.

Regardless, if you have additional bug fixes that are unrelated to the Artifactory functionality, please put that stuff in a different PR.  This PR should *only* contain the Artifactory functionality.